sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image-data.js

```diff
@@ -118,18 +144,20 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
```

Comment:
No, the test isn't drawing into a RED_INTEGER texture, it's sampling from it. The framebuffer is RGBA8 in all the tests.